test/common/base.hpp

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright (C) 2023-2024 Intel Corporation
+ * Copyright (C) 2023-2025 Intel Corporation
```

Comment:
Is it needed? there are no any other changes in this file.

Reply:
right, reverted